benches/bench_synth.rs

```diff
@@ -0,0 +1,86 @@
+use criterion::Criterion;
+use tailcall::core::blueprint::Blueprint;
+use tailcall::core::config::{Config, ConfigModule};
+use tailcall::core::ir::{Builder, Data, FieldId, Store, Synth};
+use tailcall::core::valid::Validator;
+
+const POSTS: &str = r#"
+        [
+                {
+                    "id": 1,
+                    "userId": 1,
+                    "title": "Some Title"
+                },
+                {
+                    "id": 2,
+                    "userId": 1,
+                    "title": "Not Some Title"
+                }
+        ]
+    "#;
+
+const USER1: &str = r#"
+        {
+                "id": 1,
+                "name": "foo"
+        }
+    "#;
+
+const USER2: &str = r#"
+        {
+                "id": 2,
+                "name": "bar"
+        }
+    "#;
+
+pub(crate) enum TestData {
+    Posts,
```

Comment:
use `include_str` and embed actual JSON of the `/user` API (For ALL the 10 users)